grammars/level1.lark

```diff
@@ -58,9 +58,10 @@ textwithoutspaces: /([^\n #]+)/ -> text //anything can be parsed except for a ne
 // FH Sept 2021: More info on this variable format: https://www.unicode.org/reports/tr31/tr31-1.html
 // Exact grammar stolen from: https://lark-parser.readthedocs.io/en/latest/classes.html
 
-NAME: ID_START ID_CONTINUE*
-ID_START: /[\p{Lu}\p{Ll}\p{Lt}\p{Lm}\p{Lo}\p{Nl}_]+/
-ID_CONTINUE: ID_START | /[\p{Mn}\p{Mc}\p{Nd}\p{Pc}·]+/
+NAME: LETTER_OR_UNDERSCORE LETTER_OR_NUMERAL*
+
+LETTER_OR_UNDERSCORE: /[\p{Lu}\p{Ll}\p{Lt}\p{Lm}\p{Lo}\p{Nl}_]+/
+LETTER_OR_NUMERAL: LETTER_OR_UNDERSCORE | /[\p{Mn}\p{Mc}\p{Nd}\p{Pc}·]+/
```

Comment:
I like how this is so much clearer now! I was really puzzled the first time I saw this regex and didn't understand the meaning of it!

Reply:
Yeah I just copied the names from the Python definitions but upon revisiting them and being confused 3 times myself I thought: why not make this better for us!